frontend/packages/kubevirt-plugin/src/components/modals/scheduling-modals/shared/LabelsList/labels-list.scss

```diff
@@ -0,0 +1,16 @@
+.kv-labels-list__column-title {
```

Comment:
same: do we have to use css here? can't we rather use https://patternfly-react.surge.sh/documentation/react/layouts/grid or https://patternfly-react.surge.sh/documentation/react/layouts/split or some other layouts

Reply:
ya I agree, this doesn't feel like something that needs custom css. I think we've already build very similar layouts in the console already like this right?